05/mobile.css

```diff
@@ -0,0 +1,35 @@
+.green {
+    outline: 3px solid green;
+    width: 100%;
+    height: 500px;
+}
+
+.blue {
+    outline: 2px solid #39A8F3;
+    max-width: 1000px;
+    margin: 10px auto;
+    height: 80px;
+    display: flex;
+}
+
+.yellow {
+    outline: 2px solid orange;
+    width: 20%;
+    height: 20%;
```

Comment:
Jeśli zastosojesz moje rozwiązanie to tutaj nie może być `%` bo wysokość rodzica zależy od dziecka